src/PhpDoc/LattePhpDoc.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Efabrica\PHPStanLatte\PhpDoc;
+
+final class LattePhpDoc {
```

Comment:
please fix codestyle, { to new line etc

Reply:
I will I did not care about code style yet it was just proof of concept at this point.